src/contrib/cluster/Akka.Cluster.Tools.Tests/ClusterClient/ClusterClientConfigSpec.cs

```diff
@@ -42,7 +42,7 @@ public void ClusterClientSettings_must_have_default_config()
         }
 
         [Theory]
-        [InlineData(0)]
+        [InlineData(-1)]
         [InlineData(10001)]
         public void ClusterClientSettings_must_throw_exception_on_wrong_buffer(int bufferSize)
```

Comment:
Need to add a separate spec that verifies that a 0 setting works and `ClusterClient` can start correctly, in addition to this one.